src/components/utils/story.js

```diff
@@ -0,0 +1,47 @@
+import get from "lodash/get";
```

Comment:
Any export from this file doesn't seem to be used anywhere

Reply:
not being imported any where so removed the file